pypeit/manual_extract.py

```diff
@@ -85,15 +85,15 @@ def by_fitstbl_input(cls, frame: str, inp: str, spectrograph):
         m_es = inp.split(';')
         for m_e in m_es:
             parse = m_e.split(':')
-            # det_strip will be a list of a single number (no mosaic) or 2 numbers (mosaic)
+            # det_strip will be a list of a single number (no mosaic) or >= 2 numbers (mosaic)
```

Comment:
If len(det_strip) == 1 the it is not a mosaic, otherwise it is

Just for the comment. In this way it is explicit